{"uuid": "95e8c97e-d99b-47eb-ab5d-ac130c9e3078", "children": ["0feb865e-e6f5-4d6a-8c10-1a9c7f4045bb", "336cc7e5-4001-4a48-bf7b-780fb37b1a04"], "befores": [{"name": "firmware_upgrade", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/conftest.py\", line 664, in firmware_upgrade\n    upgrade_status = fixtures_ver.setup_firmware(get_apnos, get_configuration)\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 155, in setup_firmware\n    current_version = str(ap_version).split(\"/\")[1].replace(\" \", \"\").splitlines()[0]\n"}, "attachments": [{"name": "firmwares/limit=10000&deviceType=cig_wf194c&latestonly=offset=3000", "source": "88d8e193-6eda-442d-8e83-7b8e1dd6d416-attachment.json", "type": "application/json"}, {"name": "Target firmware : ", "source": "846ab0be-7b6f-4390-87ec-554ebcda5d9d-attachment.attach"}], "start": 1638876737556, "stop": 1638876868214}], "start": 1638876737556, "stop": 1638880123029}